tensorflow/compiler/xla/mlir_hlo/lhlo_gpu/IR/lhlo_gpu_ops.td

```diff
@@ -121,6 +121,19 @@ def LHLOGPU_ConvForwardFusedSideInputOp :
          F64Attr:$side_input_scale)>.attributes);
 }
 
+def LHLOGPU_ConvForwardFusedLeakyReluAlphaOp : LHLOGPU_ConvBaseOp<"conv_forward_fused_with_leakyrelu_alpha"> {
+  let arguments = !con(
+    (ins
+       Arg<LHLO_Buffer, "", [MemRead]>:$input,
+       Arg<LHLO_Buffer, "", [MemRead]>:$filter,
+       Arg<LHLO_Buffer, "", [MemRead]>:$bias,
+       Arg<LHLO_Buffer, "", [MemWrite]>:$output,
+       Arg<LHLO_Buffer, "", [MemWrite]>:$scratch),
+     GpuConvolutionAttributes<(ins
+         ActivationAttr:$activation_mode,
+         F64Attr:$leakyrelu_alpha)>.attributes);
+}
```

Comment:
Maybe we can avoid to propose this new LHLO op and reuse the existing ConvForwardFusedOp by adding the `leakyrelu_alpha` attr. Then, for the other lowering/translation we can simply pass the 0.0 for non-leakyrelu conv.